website/docs/d/kubernetes_cluster.html.markdown

```diff
@@ -45,6 +49,10 @@ The following attributes are exported:
 
 * `fqdn` - The FQDN of the Azure Kubernetes Managed Cluster.
 
+* `private_fqdn` - The FQDN when private link has been enabled, this name is only resolvable inside the AKS vnet. 
```

Comment:
could we spell this out so it's clearer:

```suggestion
* `private_fqdn` - The FQDN of this Kubernetes Cluster when private link has been enabled. This name is only resolvable inside the Virtual Network where the Azure Kubernetes Service is located.
```

Reply:
Resolved in new commit